pythonwhat/State.py

```diff
@@ -55,62 +52,12 @@ def __init__(self, **kwargs):
         self.fun_usage = None
         self.manual_sigs = None
```

Comment:
Come to think of it: in your previous PR, you transported `converters` to the class level. Can you also do this for `manual_sigs`?